diff --git a/norec4dna/rules/FastDNARules.py b/norec4dna/rules/FastDNARules.py
index c2d1635..458bdb5 100644
--- a/norec4dna/rules/FastDNARules.py
+++ b/norec4dna/rules/FastDNARules.py
@@ -85,12 +85,12 @@ class FastDNARules:
                 # FastDNARules.g_permutation,
                 # FastDNARules.dinucleotid_runs,
                 # FastDNARules.homopolymers,
-                partial(FastDNARules.homopolymers, probs=three_strict_homopolymers()),
-                # FastDNARules.overall_gc_content,
+                partial(FastDNARules.homopolymers, probs=strict_homopolymers()),
+                FastDNARules.overall_gc_content,
                 # To change the GC error function:
-                partial(FastDNARules.overall_gc_content, calc_func=fs_gc_error_calculation),
-                # FastDNARules.windowed_gc_content,
-                partial(FastDNARules.windowed_gc_content, calc_func=ts_gc_error_calculation),
+                #partial(FastDNARules.overall_gc_content, calc_func=fs_gc_error_calculation),
+                FastDNARules.windowed_gc_content,
+                #partial(FastDNARules.windowed_gc_content, calc_func=ts_gc_error_calculation),
                 #  FastDNARules.long_strands,
                 #  FastDNARules.illegal_symbols,
                 # FastDNARules.trinucleotid_runs,
